cmd/preview/list.go

```diff
@@ -73,15 +88,66 @@ func executeListPreviews(ctx context.Context, opts ListFlags) error {
 		return fmt.Errorf("failed to get preview environments: %s", err)
 	}
 	w := tabwriter.NewWriter(os.Stdout, 1, 1, 2, ' ', 0)
```

Comment:
do we need a new instance of `tabwriter` when we output JSON/YAML?

Reply:
Probably no, should i put this in default case?